specs/binding-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/kafka/internal/KafkaFunctionsTest.java

```diff
@@ -440,6 +441,7 @@ public void shouldGenerateMergedDataExtension()
                                      .typeId(0x01)
                                      .merged()
                                          .timestamp(12345678L)
+                                         .filters(0L)
```

Comment:
The example should probably show a non-zero filter mask, because filter mask zero means the message didn't match and therefore wouldn't be delivered.